src/wallet/rpcwallet.cpp

```diff
@@ -2260,13 +2261,14 @@ UniValue z_listunspent(const UniValue& params, bool fHelp)
             "      \"address\"     (string) zaddr\n"
             "      ,...\n"
             "    ]\n"
-            "\nResult\n"
+            "\nResult (only one of the output indices will be present):\n"
```

Comment:
```suggestion
            "\nResult (only output indices for one protocol will be present):\n"
```
In the case of a Sprout output, both `jsindex` and `jsoutindex` will be present.

Reply:
Would this be better: "output indices for only one protocol will be present" (since "only" applies to "one protocol", perhaps we should place them close together)?